packages/ozone/src/db/migrations/20250923T000000000Z-scheduled-actions.ts

```diff
@@ -0,0 +1,62 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('scheduled_action')
+    .addColumn('id', 'bigserial', (col) => col.primaryKey())
+    .addColumn('action', 'varchar', (col) => col.notNull())
+    .addColumn('eventData', 'jsonb')
+    .addColumn('did', 'varchar', (col) => col.notNull())
+    .addColumn('executeAt', 'timestamptz')
+    .addColumn('executeAfter', 'timestamptz')
+    .addColumn('executeUntil', 'timestamptz')
+    .addColumn('randomizeExecution', 'boolean', (col) => col.defaultTo(false))
```

Comment:
I forget, do you still need to use `notNull()` to ensure non-nullable even when using `defaultTo()`?  I think you might.